modules/virt-querying-the-node-exporter-service-for-metrics.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-querying-the-node-exporter-service-for-metrics-_{context}"]
+= Querying the node-exporter service for metrics
+
+Metrics are exposed for virtual machines through an HTTP service endpoint under the `/metrics` canonical name. When you query for metrics, Prometheus directly scrapes the metrics from the metrics endpoint exposed by the virtual machines and presents these metrics for viewing.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges or the `monitoring-edit` role.
+* You have enabled monitoring for the user-defined project by configuring the `node-exporter` service.
+* You configured the `ServiceMonitor` resource that scrapes the metrics exposed by the `node-exporter` service.
+
+.Procedure
+* To list all available metrics for the `node-exporter` service, use the `curl` query against `http://<endpoint>/metrics`.
++
+[source,terminal]
+----
+$ curl http://<172.30.226.9100>/metrics | grep -vE "^#|^$"
```

Comment:
Change to `$ curl http://<172.30.226.9100>/metrics` --> `$ curl http://<172.30.226.162:9100>/metrics`

Reply:
If we change the command syntax for curl, would the output example that I provided be the same? @jobbler - please respond.

Reply:
No @lmandavi it is just IP address which is using port `9100`. You can see same IP at this line: https://github.com/openshift/openshift-docs/pull/41475/files#diff-7781a006fafb66cb338604e618e121a20a7310d45f45d1af9745d11626086fbbR53

Reply:
Above explanation applies to above similar comment.

Reply:
@lmandavi You have two different IP addresses and they should be the same as @sbulage points out.

Reply:
IP address has been corrected. Thanks for the clarification.